bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -283,6 +283,13 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                concord::crypto::SignatureAlgorithm::EdDSA,
                "A flag to specify the operator message signing algorithm. It is defaulted to use EDDSA algo.");
 
+  CONFIG_PARAM(keysHistoryMaxBlocksNum,
+               std::uint64_t,
+               1024u,  // TODO (efrat): default value TBD
+               "Max number of blocks to be saved in keys history column family");
```

Comment:
the description is inaccurate, should be something like, "threshold for pruning from keys history, versions below latest_block_id - this_value can be pruned"

Reply:
ok, thanks